centrifuge/src/main/java/io/github/centrifugal/centrifuge/Client.java

```diff
@@ -17,6 +17,8 @@
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 
+import javax.annotation.Nullable;
```

Comment:
Is this import available everywhere?

Reply:
I'm not sure I understand the question. I'm assuming it won't be built if it doesn't supported by some of the target platforms?

Reply:
It's part of `com.google.code.findbugs:jsr305` too